### PR TITLE
refactor: Removed hardcoded links from LoginActivity.kt file

### LINKED ISSUE #4816: Remove hardcoded links
- [ ] Login activity
https://github.com/openfoodfacts/openfoodfacts-androidapp/blob/e35897fa5a124ea68f1950d386d8996c411dd159/app/src/main/java/openfoodfacts/github/scrachx/openfood/features/login/LoginActivity.kt#L69-L70

### PR DESCRIPTION
**Pull request by pratyaksh1610**
####  Description

Removed hardcoded links from [LoginActivity.kt](https://github.com/openfoodfacts/openfoodfacts-androidapp/blob/e35897fa5a124ea68f1950d386d8996c411dd159/app/src/main/java/openfoodfacts/github/scrachx/openfood/features/login/LoginActivity.kt#L69-L70) file and added to strings.xml file

#### Related issues
Fixes #4816

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -300,7 +300,8 @@
     <string name="error_invalid_password">The password needs to be at least 6 characters long.</string>
     <string name="user">user</string>
     <string name="error_field_required">This field is required.</string>
-
+    <string name="user_login_url">%s\cgi/user.pl</string>
```
Comment (VaiTon) [inline app/src/main/res/values/strings.xml]: We shoud add `translatable="false"`

https://stackoverflow.com/questions/3925072/what-is-the-purpose-of-using-translatable-in-android-strings

**Review by VaiTon — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (pratyaksh1610) [reply app/src/main/res/values/strings.xml]: Okay,I am doing it now

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -300,7 +300,8 @@
     <string name="error_invalid_password">The password needs to be at least 6 characters long.</string>
     <string name="user">user</string>
     <string name="error_field_required">This field is required.</string>
-
+    <string name="user_login_url" translatable="false">%s\cgi/user.pl</string>
+    <string name="reset_password_url">%s\cgi/reset_password.pl</string>
```
Comment (VaiTon) [inline app/src/main/res/values/strings.xml]: Same here, sorry! @pratyaksh1610

Comment (pratyaksh1610) [reply app/src/main/res/values/strings.xml]: Okay, no problem.

Comment (pratyaksh1610) [reply app/src/main/res/values/strings.xml]: @VaiTon Made the required changes.
Please review.
Thanks.

Comment (pratyaksh1610) [reply app/src/main/res/values/strings.xml]: @VaiTon I have changed the PR title to pass all tests.

**Review by VaiTon — APPROVED** [review]

**Comment by VaiTon** [comment]
Thank you for your PR! :tada:

**Pull request merged**